packages/vuetify/src/composables/lazy.ts

```diff
@@ -0,0 +1,19 @@
+// Utilities
+import { computed, ref, watch } from 'vue'
+
+// Types
+import type { Ref } from 'vue'
+import { propsFactory } from '@/util'
+
+export const makeLazyProps = propsFactory({
+  eager: Boolean,
+}, 'lazy')
+
+export function useLazy (props: { eager: boolean }, active: Ref<boolean>) {
```

Comment:
Probably replace `useBooted` in VOverlay with this.

Reply:
And delete bootable

Reply:
I removed bootable but have not touched VOverlay. I wasn't sure what you meant by comment below, if the current `onAfterLeave` is broken or not?

Reply:
I thought I'd broken it because there isn't anything in useBooted that sets it back to false, then I realised it's actually in an external event handler. Ideally all the lazy components should have the same behaviour, so `onAfterLeave` should maybe be part of the composable instead.

Reply:
gotcha